.github/workflows/rust.yml

```diff
@@ -60,6 +60,10 @@ jobs:
             - name: Cargo check all targets.
               run: cargo check --all-targets
 
+            # A basic check for WASM target specifically
+            - name: Cargo check WASM target
+              run: cargo check --no-default-features --features jsonrpsee,web --target wasm32-unknown-unknown
```

Comment:
This will need to be in a separate wasm job so that the `wasm32-unknown-unknown` target can be installed (I checked recently and you can't specify multi ple targets in the toolchain action above at the mo :(

eg, I added this in the parachain-example PR so you could copy and tweak this:

```
    wasm_check:
        name: Cargo check (WASM)
        runs-on: ubuntu-latest
        steps:
            - name: Checkout sources
              uses: actions/checkout@v3

            - name: Install Rust stable toolchain
              uses: actions-rs/toolchain@v1
              with:
                  profile: minimal
                  toolchain: stable
                  target: wasm32-unknown-unknown
                  override: true

            - name: Rust Cache
              uses: Swatinem/rust-cache@dd05243424bd5c0e585e4b55eb2d7615cdd32f1f # v2.5.1

            # Check WASM examples, which aren't a part of the workspace and so are otherwise missed:
            - name: Cargo check WASM examples
              run: |
                cargo check --manifest-path examples/wasm-example/Cargo.toml
```